ex2.py

```diff
@@ -0,0 +1,16 @@
+from random import randint
+
+a = randint(0, 10)
+nb_coup = 0
+print('Le nombre mystère est une valeur entre 0 et 10. Entrez votre valeur :')
+
+val = 101
+while val != a :
+	val = int(input())
```

Comment:
What if input returns a string that can't be converted to int? A program should never crash in an unexpected way